### PR TITLE
Change bulkload bufsize for row-based parser

### PR DESCRIPTION
**Pull request by yhmo**
Signed-off-by: groot <yihua.mo@zilliz.com>

#17425

### CODE REVIEW

File: internal/datanode/data_node.go [hunk 1]
```diff
@@ -656,6 +671,11 @@ func (node *DataNode) Stop() error {
 	node.cancel()
 	node.flowgraphManager.dropAll()
 
+	if node.idAllocator != nil {
+		node.idAllocator.Close()
```
Comment (soothing-rain) [inline internal/datanode/data_node.go:675]: nit: log before close

**Comment by yhmo** [comment]
rerun ut

**Review by congqixia — APPROVED** [review]
/lgtm

**Pull request merged**